monitoring.yml

```diff
@@ -0,0 +1,3 @@
+- hosts: all, localhost
+  roles:
+    - { role: monitoring, when: setupMonitoring == True }
```

Comment:
Can we keep the yaml syntax the consistent

Reply:
Should be fixed in latest update